### ISSUE TITLE
Building blueprints appear at different heights when they are unable to be built inside the map

### ISSUE THREAD

**Issue by mercurialisbetter** [tf2classic/tf2classic-issue-tracker#209]
As shown in the video below, the Sentry Gun will appear below the floor, the Dispenser slightly above it, and the Teleporters and Jump Pads significantly above it. Expected behaviour is that the height is the same for all blueprints.

https://github.com/tf2classic/tf2classic-issue-tracker/assets/113199408/2ab24996-9721-4bbe-abaa-33e7a501d4f5

**Comment by Trotim** [comment]
Is this the same in live TF2?

**Comment by mercurialisbetter** [comment]
Yes, but the heights are slightly different. The Sentry Gun blueprints do not phase through the floor. Also, the issue where the teleporter builds slightly farther away is still present in live TF2.
